### ISSUE TITLE
no option to open bookmark in Tor Window

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#1699]
## Description 
Found while testing https://github.com/brave/brave-browser/issues/1387

Per description in 1387 it seems as though when on Bookmarks Manager, if you right click on a bookmark or click on the 3 dots menu for a bookmark, there should be an option to 'Open Private Window with Tor' (or similar wording). This option does not exist.

## Steps to Reproduce

   1. If you don't have bookmarks, add at least one.
   2. Navigate to Bookmark Manager.
   3. Click on 3 dots menu for bookmark or right click on bookmark.

## Actual result:
No option to open in Tor window from Bookmarks manager or from Bookmarks Bar:
![screen shot 2018-10-18 at 10 05 45 am](https://user-images.githubusercontent.com/28145373/47160192-67631a00-d2bd-11e8-9f9a-8ef1b386b3c0.png)

![screen shot 2018-10-18 at 10 07 10 am](https://user-images.githubusercontent.com/28145373/47160323-aabd8880-d2bd-11e8-9a87-01bf57b801dd.png)

## Expected result:
Option to open in Tor window.

## Reproduces how often: 
easy

## Brave version (brave://version info)

Brave | 0.55.17 Chromium: 70.0.3538.67 (Official Build) (64-bit)
-- | --
Revision | 9ab0cfab84ded083718d3a4ff830726efd38869f-refs/branch-heads/3538@{#1002}
OS | Mac OS X

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? yes
- Does it reproduce on browser-laptop? n/a

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? n/a
- Is the issue reproducible on the latest version of Chrome? n/a

### Additional Information
Reproduced by @btlechowski and @srirambv on Win/Linux

**Comment by LaurenWags** [comment]
cc @tomlowenthal - the way I read https://github.com/brave/brave-browser/issues/1387 this should have been included (3 from description) but if this is unnecessary please advise.

**Comment by tomlowenthal** [comment]
Good catch, you're right. Everywhere Chromium has open in new tab / window / incognito window we should add Tor windows.

**Comment by riastradh-brave** [comment]
Grah, I thought I'd dealt with that one.  Apparently I didn't.

**Comment by simonhong** [comment]
I think the reason is tor window is based on guest profile. @darkdh

**Comment by NumDeP** [comment]
I'd like it if the labelling bookmarks feature in bookmarks of differentiating all the window types the bookmarks were saved under were developed alongside this and this #910 so users don't have to recall which window or tab type to open it under and so Brave can either prevent or at least warn a user if a tab is being opened under an unusual window type.

**Comment by riastradh-brave** [comment]
This is blocked on me figuring out which of the multiple bookmark manager UI mechanisms need to be updated; as I recall the draft I started a while back (https://github.com/brave/brave-core/tree/riastradh-1699-openbookmarktor) was in the wrong place.

**Comment by DrEVILish** [comment]
Currently still an issue one the Lastest version:

I'd like to also add a request for there to be an option on a bookmark to always open in TOR or to have a TOR specific url.
